<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1089</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainMoil</string>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>241</width>
      <height>141</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>241</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label3">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>241</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>241</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label5">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>180</y>
      <width>241</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label6">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>180</y>
      <width>241</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>100</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox01">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>350</y>
      <width>611</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton_play">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>2</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_left">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>2</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_right">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>2</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_top">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>2</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox03">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>140</y>
      <width>131</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="ap_up">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="ap_right">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="ap_left">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="ap_reset">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="ap_down">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>480</y>
      <width>590</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="time1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>450</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="time2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>440</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>00:00</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox04">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>120</y>
      <width>131</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="ap_DBG_LL">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>110</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>LL</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ap_DBG_R">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>R</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ap_DBG_L">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>L</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ap_DBG_F">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>F</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ap_DBG_D">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>D</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ap_DBG_LR">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>110</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>LR</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Center">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>31</width>
       <height>55</height>
      </rect>
     </property>
     <property name="text">
      <string>Set</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Cx">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>10</y>
       <width>60</width>
       <height>25</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>12</y>
       <width>21</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Cx</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Cy">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>40</y>
       <width>60</width>
       <height>25</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>42</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Cy</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1089</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMainMoil">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionLoad_Image"/>
    <addaction name="actionLoad_Video"/>
    <addaction name="actionLoad_Note"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionMOIL"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionCamera_URL"/>
    <addaction name="actionGenerate_Cache"/>
   </widget>
   <addaction name="menuMainMoil"/>
   <addaction name="menuSettings"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoad">
   <property name="text">
    <string>Load Parameter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionLoad_Image">
   <property name="text">
    <string>Load Image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionMulti_view">
   <property name="text">
    <string>Multi-view</string>
   </property>
  </action>
  <action name="actionMOIL">
   <property name="text">
    <string>MOIL</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionCH1">
   <property name="text">
    <string>CH1</string>
   </property>
  </action>
  <action name="actionCH2">
   <property name="text">
    <string>CH2</string>
   </property>
  </action>
  <action name="actionCH3">
   <property name="text">
    <string>CH3</string>
   </property>
  </action>
  <action name="actionCH4">
   <property name="text">
    <string>CH4</string>
   </property>
  </action>
  <action name="actionCH5">
   <property name="text">
    <string>CH5</string>
   </property>
  </action>
  <action name="actionCH6">
   <property name="text">
    <string>CH6</string>
   </property>
  </action>
  <action name="actionVideo_Source">
   <property name="text">
    <string>Video Source</string>
   </property>
  </action>
  <action name="actionLoad_Video">
   <property name="text">
    <string>Load Video</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionCamera_URL">
   <property name="text">
    <string>Camera URL</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionLoad_Note">
   <property name="text">
    <string>Load Note</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionGenerate_Cache">
   <property name="text">
    <string>Generate Cache</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
